python-packages/fle_utils/django_utils/command.py

```diff
@@ -51,36 +50,6 @@ def call_command_with_output(cmd, *args, **kwargs):
         sys.exit   = backups[2]
 
 
-def call_command_subprocess(cmd, *args, **kwargs):
```

Comment:
This is safe, the function was re-declared further down in the original, so this code block was always dead.